package.json

```diff
@@ -19,8 +19,7 @@
     "builder:check": "eslint config",
     "test-func-remote": "TEST_FUNC=remote mocha --opts ./test/func/mocha.opts",
     "test-func-static": "TEST_FUNC=static mocha --opts ./test/func/mocha.opts",
-    "test-func-dev": "TEST_FUNC=dev mocha --opts ./test/func/mocha.opts",
-    "install-selenium": "selenium-standalone install"
```

Comment:
I'd love some feedback on something I'm trying here:

In the previous commit I added `selenium-standalone install` as an npm `postinstall` script to `builder-docs-archetype-dev`, where I also added a `spec-helper.js` that sets up Rowdy (thus starting selenium) and all the mocha before/after/etc hooks. `selenium standalone install` needed to inside the builder-docs-archetype-dev folder in order for it to be found. 

My goal is to keep all the mocha hooks in the archetype, since the 4 downstream docs site all use the same webpack/static server configs.